src/assets/wise5/authoringTool/node/node-authoring/node-authoring.component.ts

```diff
@@ -352,13 +320,13 @@ class NodeAuthoringController {
      * take effect. If we don't use a timeout, the user won't see any change
      * in the UI.
      */
-    this.$timeout(() => {
+    setTimeout(() => {
```

Comment:
Is this timeout still needed? Deleting component(s) seems to work without the timeout for me.
```suggestion
```